server/app/services/Path.java

```diff
@@ -118,6 +119,27 @@ public Path join(String path) {
    */
   public Path join(Scalar scalar) {
     Path other = Path.create(scalar.name());
+    return join(other);
+  }
+
+  /**
+   * Append the provided {@link ApiPathSegment} to this {@link Path}.
```

Comment:
**thought:** Would be nice to make the other join functions javadocs consistent (can be here or in a separate PR).

Reply:
Done!